Makefile.common

```diff
@@ -169,9 +169,6 @@ common-vet:
 common-lint: $(GOLANGCI_LINT)
 ifdef GOLANGCI_LINT
 	@echo ">> running golangci-lint"
-# 'go list' needs to be executed before staticcheck to prepopulate the modules cache.
-# Otherwise staticcheck might fail randomly for some reason not yet explained.
-	$(GO) list -e -compiled -test=true -export=false -deps=true -find=false -tags= -- ./... > /dev/null
```

Comment:
Why removing this?

Reply:
It looks like this is a remnant from 4 years ago when golangci-lint was added that isn't necessary to make it work anymore?